### PR TITLE
Update changelog 30-April-2021

### PR DESCRIPTION
**Pull request by danimtb**

### CODE REVIEW

**Review by jgsogo — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM

**Review by SSE4 — APPROVED** [review]

File: docs/changelog.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Changelog
 
-### 08-April-2021 - 17:01 CET
+### 30-April-2021 - 13:52 CEST
+
+- [feature] Add mark in logs to indicate output has been trimmed.
+- [feature] Stop uploading packages to Bintray.
+- [job] RequestReviews: Automatically request team reviews for PRs ready for review.
```
Comment (prince-chrismc) [inline docs/changelog.md:7]: I am super curious, how did you guys define "ready for review"?

Comment (SSE4) [reply docs/changelog.md:7]: same as described in [docs](https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md#automatic-merges) minus reviews, so effectively:
- Author is an added beta user.
- Author has signed CLA.
- PR is not a Draft or WIP.
- PR has a green status (successful build).
- PR doesn't have merge conflicts with master branch.

Comment (prince-chrismc) [reply docs/changelog.md:7]: Thanks!

**Pull request merged**